python/tank/util/qt_importer.py

```diff
@@ -163,8 +163,6 @@ def _import_module_by_name(self, parent_module_name, module_name):
             logger.debug("Unable to import module '%s': %s", module_name, e)
             pass
         return module
-
-    def _import_pyside(self):
```

Comment:
I think this line was removed by accident. Can you double check?

Reply:
No, actually, this line should be removed along with the rest of the function. We don't want to import pyside anymore.

Reply:
But the rest of the function doesn't seem deleted, am I missing something? @eduardoChaucaGallegos

Reply:
you are right, rest of the function was not deleted and not given an error because that code was after a return statement as part of function "_import_module_by_name", but now it is deleted